src/tracepoint.jl

```diff
@@ -124,58 +124,97 @@ function _tracepoint_func(name::Union{String, Nothing}, ex::Expr, mod::Module, s
     return cdef
 end
 
+root_module(m::Module) = parentmodule(m) === m ? m : root_module(parentmodule(m))
+
+
+# Example (Local)Preferences.toml:
+#
+# [MyPkg.Tracy]
+# enabled = true
+# whitelist = ['foo', 'bar.*']
+# blacklist = []
+function get_preferences(mod::Module)
```

Comment:
I don't really understand why we need to do all this over `@load_preferences`. What do we do that is so unique?